e2e/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM cypress/included:10.10.0
 
 WORKDIR /opt
-RUN npm install cypress-file-upload cypress-pseudo-localization dotenv
+RUN npm i
```

Comment:
This assumes the existence of a `package.json` file to get the dependencies from